x/staking/src/message.rs

```diff
@@ -10,7 +10,7 @@ use serde::Serialize;
 #[serde(tag = "@type")]
 #[allow(clippy::large_enum_variant)]
 pub enum Message {
-    #[serde(rename = "/cosmos.staking.v1beta1.CreateValidator")]
+    #[serde(rename = "/cosmos.staking.v1beta1/MsgCreateValidator")]
```

Comment:
Why the extra `/`

Reply:
My mistake. I thought that it same as for query